src/storage/src/frontend/tests/mod.rs

```diff
@@ -69,12 +69,19 @@ fn insert_into<P: backend::BackendStorage>(
     schema_name: &str,
     table_name: &str,
     values: Vec<&str>,
+    columns: Option<Vec<&str>>,
 ) {
+    let columns = match columns {
+        Some(cols) => Some(cols.into_iter().map(ToOwned::to_owned).collect()),
+        None => None,
+    };
```

Comment:
Consider, using `Optional::map` here, like:
```rust
columns.map(|cols| cols.into_iter().map(ToOwned::to_owned).collect())
```

Reply:
`Option<Vec<String>>` is replaced with `Vec<String>`. So it is resolved.